bxserum/provider/ws.go

```diff
@@ -356,6 +356,43 @@ func (w *WSClient) SubmitCancelByClientOrderID(
 	return w.signAndSubmit(order.Transaction, skipPreFlight)
 }
 
+func (w *WSClient) PostCancelAll(market, owner string, openOrdersAddresses []string) (*pb.PostCancelAllResponse, error) {
+	request, err := w.jsonRPCRequest("PostCancelAll", &pb.PostCancelAllRequest{
```

Comment:
have you merged from develop? this look out of date

Reply:
Which part looks out of date? The `PostCancelAll` method?

Reply:
yeah, I deleted `jsonRPCRequest
<img width="984" alt="Screen Shot 2022-07-21 at 5 23 16 PM" src="https://user-images.githubusercontent.com/5728920/180325055-a2809df9-bb92-4a63-a31f-f4a56d7dcc6b.png">
`